docs/native/api.md

```diff
@@ -132,11 +132,17 @@ match transaction {
 
 ## transaction\_sign\_raw
 
-Sign a transaction and return a raw signature (RSV format).
+Sign a transaction and return a raw signature. Now support `Secp256k1` signing (RSV format) and `BLS` signing. The type of signature chosen will be dictated by the protocol of the `from` field of the transaction.
+
+e.g :
+
+* "t**1**b4zd6ryj5dsnwda5jtjxj6ptkia5e35s52ox7ka" is a [protocol 1 address](https://filecoin-project.github.io/specs/#protocol-1-libsecpk1-elliptic-curve-public-keys) therefore `transaction_sign_raw` will attempt to the sign the transaction using `Secp256k1`.
+
+* "t**3**vxrizeiel2e2bxg3jhk62dlcutyc26fjnw6ua2sptu32dtjpwxbjawg666nqdngrkvvn45h7yb4qiya6ls7q" is a [protocol 3 address](https://filecoin-project.github.io/specs/#protocol-3-bls). Here we will use `BLS` signing scheme.
 
 Arguments:
 * **transaction**: a filecoin transaction;
-* **privatekey**: a private key as hexstring;
+* **privatekey**: a private key as hexstring (should be the associated private key of `from` address of the transaction);
```

Comment:
It is a bit strange to require hexstring in the native rust API but we can later improve this in another issue.

Reply:
Good catch. It is a mistake in the doc. We are requiring bytes array in the code.